src/ui/marker.js

```diff
@@ -159,6 +158,7 @@ export default class Marker extends Evented {
             this._offset = Point.convert(options && options.offset || [0, 0]);
         }
 
+        if (!this._element.hasAttribute('aria-label')) this._element.setAttribute('aria-label', 'Map marker');
```

Comment:
IMO "map marker" seems a bit verbose, "marker" alone should be sufficient. `aria-label="map"` already provides the context for the markers (although I guess that depends on https://github.com/mapbox/mapbox-gl-js/issues/11038 to some degree).